.travis.yml

```diff
@@ -0,0 +1,69 @@
+language: cpp
+
+script:
+ - "bash -ex .travis-ci.sh"
+
+matrix:
+  fast_finish: true
+  include:
+    - compiler: clang
```

Comment:
We can drop the double compiler check. Since this will only be run on developer machines, we have far less combinations to tests.